bci_tester/fips.py

```diff
@@ -64,6 +64,39 @@
 
 assert len(set(ALL_DIGESTS)) == len(ALL_DIGESTS)
 
+#: gcrypt digests that are not FIPS compliant
+NONFIPS_GCRYPT_DIGESTS: Tuple[str] = (
+    "sm3",
+    "md5",
+    "ripemd160",
+    "tiger",
+    "tiger2",
+    "tiger192",
+    "md4",
+    "whirlpool",
+    "gostr3411_94",
+    "stribog256",
+    "stribog512",
+)
+
+#: FIPS compliant gcrypt digests
+FIPS_GCRYPT_DIGESTS: Tuple[str] = (
+    "sha1",
+    "sha224",
+    "sha256",
+    "sha384",
+    "sha512",
+    "sha512_224",
+    "sha512_256",
+    "sha3-224",
+    "sha3-256",
+    "sha3-384",
+    "sha3-512",
+    "crc32",
+    "crc32rfc1510",
+    "crc24rfc2440",
```

Comment:
These are error correction codes, they are not really interesting in a cryptographic context 
```suggestion
```

Reply:
removed